web_ui/src/pages/user-management/personal-access-token-page/components/personal-access-tokens-menu.component.tsx

```diff
@@ -7,11 +7,20 @@ import { useUsers } from '@geti/core/src/users/hook/use-users.hook';
 import { ActionButton, Flex, Item, Menu, MenuTrigger, Text } from '@geti/ui';
 import { Delete, Edit, MoreMenu } from '@geti/ui/icons';
 import { useOverlayTriggerState } from '@react-stately/overlays';
+import { AxiosError } from 'axios';
 import dayjs from 'dayjs';
 
-import { usePersonalAccessToken } from '../../../../core/personal-access-tokens/hooks/use-personal-access-token.hook';
+import {
+    DELETE_ERROR,
+    DELETE_MESSAGE,
+    UPDATE_ERROR,
+    UPDATE_MESSAGE,
+    usePersonalAccessToken,
+} from '../../../../core/personal-access-tokens/hooks/use-personal-access-token.hook';
```

Comment:
I think this part feels counter intuitive to me here: on the one hand we want the consumer of the hook to choose what notifications to choose, but we end up with adding back tight coupling because we only use data from the hook's file.

Reply:
The component was already coupled with the hook, it wont work without it. Maybe it's because you see all message imports? I guess i could also move them here to this file, where they are used, wdyt?

Reply:
Yeah but before the notification messages and sending the notifications were both in the `usePersonalAccessToken` hook, and now we have the messages in the hook, but sending them is being done by the consumer.
This feels odd because it gives the impression that the hook intents for notifications to be send but doesn't do so.

> I guess i could also move them here to this file, where they are used, wdyt?

Yeah that would be better imho.

But I'd prefer us to implement the first point I mentioned above (not using notifications for errors and inline them instead), I think that will result in a better UX.